integration-tests/gradle/src/main/resources/basic-kotlin-application-project/build.gradle

```diff
@@ -0,0 +1,26 @@
+plugins {
+    id 'org.jetbrains.kotlin.jvm' version "1.5.30"
```

Comment:
Given that this file is a test resource, it'd be great if we could use version properties in place of specific versions and enable filtering when copying it to the build dir.

Reply:
👍 - I was surprised how many places had to be adjusted when I was involved in the latest kotlin update.

Reply:
Right, I extracted it in a the `gradle.properties`

Reply:
This doesn't really solve the issue though. It should be controlled by https://github.com/quarkusio/quarkus/blob/main/build-parent/pom.xml#L23.
It's ok to merge it. But we should still fix the versions by using the build-parent properties everywhere in the Gradle testsuite. @glefloch do you think you could look into that?

Reply:
The gradle.properties in this case is still a resource file that is processed by the resources plugin. We should simply look into enabling filtering.

Reply:
Right, this should be easier as this is now a maven project. I will go into all sample project.

Reply:
That'd be great, thanks @glefloch